site-main/variables.tf

```diff
@@ -3,6 +3,7 @@ variable region {
 }
 
 variable domain {}
+variable bucket_name {}
```

Comment:
In fact no, I tried but terraform can't have interpolation in default variable 

https://github.com/hashicorp/terraform/issues/4084